enterprise/tls-rbac-demo/README.md

```diff
@@ -0,0 +1,32 @@
+# Hazelcast Dynamic RBAC demo for TLS authentication
+
+This code sample demonstrates a dynamic reconfiguration of role-based access control in Hazelcast.
+
+The project contains three runnable classes:
+* `Server`: runs a Hazelcast server instance embedded inside a simple Java wrapper application;
+* `TimestampClient`: runs a Hazelcast client that puts and gets timestamps into an `IMap`;
+* `ReplacePermissions`: dynamically changes the RBAC configuration by starting a new lite-member with a new permissions configuration. It gets a new permission configuration XML file path as the argument.
+
+The lite-member has the `on-join-operation` attribute configured to value `SEND` to distribute the new set of permissions.
+
+```java
+  config.getSecurityConfig().setOnJoinPermissionOperation(SEND);
+```
+
+[Look into the documentation](https://docs.hazelcast.com/imdg/4.1/security/native-client-security#handling-permissions-when-a-new-member-joins) for more details about handling permissions when a new member joins.
```

Comment:
Should we use latest docs, instead IMDG 4.1 docs?

https://docs.hazelcast.com/hazelcast/latest/security/native-client-security#handling-permissions-when-a-new-member-joins

Reply:
I don't like using the `latest` as the URL could change (refactoring, chapters reordering, etc.). I'll use the 5.0 URL instead.